weaviate/collections/data.py

```diff
@@ -374,43 +414,53 @@ def replace(
 
     def update(
         self,
-        properties: Properties,
         uuid: UUID,
-        vector: Optional[List[float]] = None,
+        properties: Optional[Properties] = None,
         references: Optional[WeaviateReferences] = None,
+        vector: Optional[List[float]] = None,
     ) -> None:
         """Update an object in the collection.
 
         This is equivalent to a PATCH operation.
 
+        If the object does not exist yet, it will be created.
+
         Arguments:
-            `properties`
-                The properties of the object, REQUIRED.
             `uuid`
                 The UUID of the object, REQUIRED.
-            `vector`
-                The vector of the object.
+            `properties`
+                The properties of the object.
             `references`
                 Any references to other objects in Weaviate.
+            `vector`
+                The vector of the object.
         """
-        weaviate_obj: Dict[str, Any] = {
-            "class": self.name,
-            "properties": self._serialize(properties, references),
-        }
+        if properties is not None and not isinstance(properties, dict):
+            _raise_invalid_input("properties", properties, dict)
+        if references is not None and not isinstance(references, dict):
+            _raise_invalid_input("references", references, dict)
+
+        props, refs = {}, {}
+        if properties is not None:
+            props = self._serialize_props(properties)
+        if references is not None:
+            refs = self._serialize_refs(references)
```

Comment:
`props = self._serialize_props(properties) if properties is not None else {}`
?